src/app/sale/handlers/post.handler.ts

```diff
@@ -0,0 +1,35 @@
+// Post handler
+// Single Responsibility File, for handling requests
+import { RequestHandler, Request, Response } from 'express';
+import { makeSale } from '../sales.service';
+
+type Params = {};
+type Query = {};
+type Body = {};
+type Req = Request<Params, {}, Body, Query>;
+type Res = Response;
+
+// request handler functions array that express allows us to pass to it
+// middlewares goes here like validation and loggers
+export const handler: RequestHandler[] = [
+  // auth,
+  // logger,
+  async (req: Req, res: Res) => {
+    const saleData = {
+      customerName: 'Lin',
```

Comment:
This is hardcoded

Reply:
Sorry, yes it is, I was mimic some stuff since I haven't totally figure it put your sample of a request but I wanted to at leat finish some endpoint setting.